client/stylesheets/loading.scss

```diff
@@ -1,8 +1,11 @@
-.loading {
-  position: fixed;
+.loading-fullsize {
```

Comment:
Is there supposed to be a `loading-inline` class defined somewhere, or is the implication that `.loading-inline` as used in Loading.tsx carries no styles?

Reply:
No styling currently. Mostly wanted to preserve future optionality.

Reply:
Seems reasonable; just wanted to make sure I wasn't missing something.